### PR TITLE
Use urlsafe_base64 for url sensitive chars

### PR DESCRIPTION
**Pull request by kyuden**
We can use the urlsafe one.

### CODE REVIEW

File: lib/sorcery/model/temporary_token.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@ def self.included(base)
 
       # Random code, used for salt and temporary tokens.
       def self.generate_random_token
-        SecureRandom.base64(15).tr('+/=lIO0', 'pqrsxyz')
+        SecureRandom.urlsafe_base64(15).tr('lIO0', 'sxyz')
```
Comment (athix) [inline lib/sorcery/model/temporary_token.rb]: Using urlsafe here is definitely preferable, although I wonder if we can deprecate this model entirely by using rails built in `has_secure_token`?

**Comment by kyuden** [comment]
The behavior is the almost same around this change.
Are we supposed to need an entry on CHANGELOG.md in this case?

File: lib/sorcery/model/temporary_token.rb (hunk 1 above)
Comment (kyuden) [inline lib/sorcery/model/temporary_token.rb]: When we drop rails 4.x supports, we might be able to deprecate it with `has_secure_token`.

**Comment by athix** [comment]
Unless I'm mistaken, this changes behavior that could affect how someone handles the tokens (if it's not URL safe to begin with, someone might be running a strip method before using it in URLs). It likely won't affect most users, but I would err on the side of caution.

**Pull request merged**